tests/e2e/bento_server_http/tests/test_meta.py

```diff
@@ -26,7 +26,7 @@ async def test_api_server_meta(host: str) -> None:
     status, _, _ = await async_request("GET", f"http://{host}/docs.json")
     assert status == 200
     status, _, _ = await async_request("GET", f"http://{host}/readyz")
-    assert status == 200
+    assert status == 500
```

Comment:
This is wrong. I need to wait first for runners to get ready and not just change it to 500. Working on this fix.